// Copyright (c) 2017-2019, NVIDIA CORPORATION.  All rights reserved.

/**
* experiment.proto: Protocol buffer definition for a top-level message
* encapsulating multiple other message types to form a complete specification
* for a MakeNet experiment.
*/

syntax = "proto3";

import "nvidia_tao_tf1/cv/makenet/proto/model_config.proto";
import "nvidia_tao_tf1/cv/makenet/proto/training_config.proto";
import "nvidia_tao_tf1/cv/makenet/proto/eval_config.proto";

message Experiment {
    EvalConfig eval_config = 1;
    ModelConfig model_config = 2;
    TrainConfig train_config = 3;
}
